Library/Formula/dpkg.rb

```diff
@@ -23,30 +23,31 @@ class Dpkg < Formula
   end
 
   def install
-    # There was a commit merged prior to this release that forgot to add ; to the end of function lines
-    # Consequently = Build failure. Put the ; in here to fix the issue.
-    # Also removes a function left over from previous refactoring which causes issues now.
-    # This will all be in the 1.17.22 release.
-    # https://lists.debian.org/debian-dpkg/2014/11/msg00029.html
-    inreplace "lib/dpkg/fdio.c" do |s|
-      s.gsub! "fs_preallocate_setup(&fs, F_ALLOCATECONTIG, offset, len);", "fd_preallocate_setup(&fs, F_ALLOCATECONTIG, offset, len);"
-      s.gsub! "fs_preallocate_setup(&fs, F_ALLOCATEALL, offset, len);", "fd_preallocate_setup(&fs, F_ALLOCATEALL, offset, len);"
-      s.gsub! "rc = fcntl(fd, F_PREALLOCATE, &fs)", "rc = fcntl(fd, F_PREALLOCATE, &fs);"
-    end
-
     # We need to specify a recent gnutar, otherwise various dpkg C programs will
     # use the system "tar", which will fail because it lacks certain switches.
     ENV["TAR"] = Formula["gnu-tar"].opt_bin/"gtar"
     system "./configure", "--disable-dependency-tracking",
                           "--prefix=#{prefix}",
+                          "--sysconfdir=#{etc}",
+                          "--localstatedir=#{var}",
                           "--disable-dselect",
                           "--disable-linker-optimisations",
-                          "--disable-start-stop-daemon",
-                          "--disable-update-alternatives"
+                          "--disable-start-stop-daemon"
     system "make"
     system "make", "install"
   end
 
+  def post_install
+    unless File.exist? etc/"dpkg/origins/dummy"
+      (etc/"dpkg/origins/dummy").write <<-EOS
+Vendor: dummy
+EOS
+    end
+    unless File.exist? etc/"dpkg/origins/default"
+      ln_s "dummy", etc/"dpkg/origins/default"
+    end
+  end
```

Comment:
If I move these two things into `def install` and remove the `unless` around them, then installation fails with

    Error: Will not overwrite /usr/local/etc/dpkg/origins/dummy

if the file already exists.